OpenRA.Game/Server/Server.cs

```diff
@@ -810,18 +920,50 @@ public void StartGame()
 			if (LobbyInfo.NonBotClients.Count() == 1)
 				LobbyInfo.GlobalSettings.OrderLatency = 1;
 
+			// Enable game saves for singleplayer missions only
+			// TODO: Enable for skirmish once the AI supports state-restoration
+			// TODO: Enable for multiplayer (non-dedicated servers only) once the lobby UI has been created
+			LobbyInfo.GlobalSettings.GameSavesEnabled = !Dedicated && Map.Visibility == MapVisibility.MissionSelector &&
```

Comment:
Remove the `Map.Visibility == MapVisibility.MissionSelector` here to enable saving in skirmish.